### PR TITLE
feat: add Strategy[StrategyCurveBBTCVoterProxy]

### PR DESCRIPTION
**Pull request by orbxball**

### CODE REVIEW

File: contracts/strategies/StrategyCurveBBTCVoterProxy.sol [hunk 1]
```diff
@@ -0,0 +1,685 @@
...
+
+            address[] memory path = new address[](3);
+            path[0] = crv;
+            path[1] = weth;
+            path[2] = wbtc;
+
+            Uni(dex).swapExactTokensForTokens(_crv, uint256(0), path, address(this), now.add(1800));
+        }
+        uint256 _wbtc = IERC20(wbtc).balanceOf(address(this));
+        if (_wbtc > 0) {
```
Comment (storming0x) [inline contracts/strategies/StrategyCurveBBTCVoterProxy.sol:185]: guessing wbtc because of better liquidity in uniswap? is that the case?

**Review by storming0x — APPROVED** [review]
Diffed review against [StrategyCurveGUSDVoterProxy.sol](https://github.com/iearn-finance/yearn-protocol/blob/develop/contracts/strategies/StrategyCurveGUSDVoterProxy.sol)

Looks good apart from missing license

TODO: will do house keeping before merging

File: contracts/strategies/StrategyCurveBBTCVoterProxy.sol [hunk 2]
```diff
@@ -0,0 +1,685 @@
+pragma solidity ^0.5.17;
```
Comment (storming0x) [inline contracts/strategies/StrategyCurveBBTCVoterProxy.sol]: Missing license identifier

**Review by storming0x — CHANGES_REQUESTED** [review]
small change, needs license identifier

File: contracts/strategies/StrategyCurveBBTCVoterProxy.sol (hunk 1 above)
Comment (orbxball) [reply contracts/strategies/StrategyCurveBBTCVoterProxy.sol:185]: Yes. better liquidity on either dex.

**Review by storming0x — APPROVED** [review]

**Pull request merged**